proto/prehog/v1alpha/teleport.proto

```diff
@@ -624,6 +624,33 @@ message AssistCompletionEvent {
   int64 completion_tokens = 5;
 }
 
+// AssistExecutionEvent is an event that is emitted when an Assist command
+// execution is triggered by the user.
+message AssistExecutionEvent {
+  // anonymized
+  string user_name = 1;
+  // ConversationId is the UUID that identifies a single Assist conversation
+  string conversation_id = 2;
+  // NodeCount is the number of nodes the command was executed on
+  int64 node_count = 3;
+  // TotalTokens is the total amount of token used to generate the command summary
+  int64 total_tokens = 4;
```

Comment:
Do we need it if we already have prompt + response counts? This is always the sum of those.

Reply:
It's already the case for the existing discussion events, I would prefer being consistent.